packages/node-sdk/paima-db/src/scheduled-constructors.ts

```diff
@@ -23,14 +23,15 @@ import type {
 export function createScheduledData(
   inputData: string,
   blockHeight: number,
-  cdeName?: string,
-  txHash?: string
+  source: { cdeName: string; txHash: string; network: string } | { precompile: string }
```

Comment:
does it really make sense to expose both use-cases in a public function? I think only the engine should use the cdeName part.